server.py

```diff
@@ -108,6 +113,17 @@ async def delete_url(alias: str):
           return {"message": "URL deleted successfully"}
       else:
           raise HTTPException(status_code=HttpResponse.NOT_FOUND.code)
+      
+@app.post("/qr/{alias}") 
+async def qr(alias: str):
+    logging.debug(f"/qr code generation called with alias: {alias}")
+    with MetricsHandler.query_time.labels("qr").time():
+      maybe_image = qr_code_cache.find(alias)
+      if maybe_image is None:
+        url_output = sqlite_helpers.get_url(DATABASE_FILE, alias)
+        if url_output is None:
+            raise HTTPException(status_code=HttpResponse.NOT_FOUND.code)
+        return qr_code_cache.add(alias)
```

Comment:
i think we have to return a `FileResponse` of the qr code?
```suggestion
    with MetricsHandler.query_time.labels("qr").time():
      maybe_image = qr_code_cache.find(alias)
      if maybe_image is not None:
        return FileResponse(
          maybe_image_data,
          media_type='image/jpeg',
        )
      
      url_output = sqlite_helpers.get_url(DATABASE_FILE, alias)
      if url_output is None:
          raise HTTPException(status_code=HttpResponse.NOT_FOUND.code)
      image_data = qr_code_cache.add(alias)
      return FileResponse(
        image_data,
        media_type='image/jpeg',
      )
```


line 3 would need to become
```py
from fastapi.responses import RedirectResponse, HTMLResponse, FileResponse
```

Reply:
resolved with [6c523a1](https://github.com/SCE-Development/cleezy/pull/39/commits/6c523a1c29f1a8ea6e7fc8a524964bbf8ed825d6)